FirebaseDatabaseSwift/Sources/third_party/RTDBEncoder/RTDBEncoder.swift

```diff
@@ -0,0 +1,2585 @@
+// This file is derived from swift/stdlib/public/Darwin/Foundation/JSONEncoder.swift
```

Comment:
For future readers - I diffed this with the copy from the Swift lib and there were no significant changes.